src/Http/Controllers/ContextTypes/BaseType.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+namespace TCG\Voyager\Http\Controllers\ContentTypes;
```

Comment:
You have a typo in all of your new classes.  You used the namespace `ContentTypes`, but named the folder `ContextTypes`, so the classes aren't being found.  This is causing tests to fail.  Please fix it.

Reply:
This has been fixed